modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/components/AutoFocus.es.js

```diff
@@ -34,7 +34,7 @@ export function AutoFocus({children}) {
 				setTimeout(() => setIncrement((value) => value + 1), 5);
 			}
 			else {
-				if (!document.activeElement.id) {
+				if (!document.activeElement) {
 					const firstInput = childRef.current.querySelector('input');
 					const sidebarOpen = document.querySelector(
 						'.ddm-form-builder--sidebar-open'
```

Comment:
@georgel-pop-lr Is there any case where `document.activeElement` is undefined?

For example, I tested this bug [LPS-134049](https://issues.liferay.com/browse/LPS-134049), which is already closed, and it is happening again because the `document.activeElement` (or `document.activeElement.id`) is always `true`.

Could you check if it is possible to change this condition to meet both cases?

Thanks!

Reply:
Hi @carolmariaabb,

So from what I understand reading [LPS-134049](https://issues.liferay.com/browse/LPS-134049) the focus should only be happening in form preview mode but not on a normal page.

Is that right?

Regards.

Reply:
As I understand it, not only preview mode but also when accessing the shared url.

Reply:
Hi @georgel-pop-lr, 
I talked to our QA and another ticket, similar to [LPS-134049](https://issues.liferay.com/browse/LPS-134049), was created. The Bravo Team will work on this issue. 

Therefore, I'll approve this PR since it's enough to fix the issue described in LPS-152215.

Thanks!